client/.storybook/preview.js

```diff
@@ -1,4 +1,5 @@
+import "!style-loader!css-loader!./main.css";
```

Comment:
I guess you like to import the same styles as in your App?
Instead, you can add a [global decorator ](https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators) and import `GlobalStyles`.
You can remove the `main.css` afterwards.